kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/industrialforegoing/laser_drill_ore.js

```diff
@@ -112,20 +103,17 @@ onEvent('recipes', (event) => {
                     weight: 1
                 }
             ],
-            catalyst: {
-                item: industrialforegoing.laser_lens.gray
-            }
+            catalyst: { item: industrialforegoing.laser_lens.gray },
+            id: `${id_prefix}fluorite`
         },
         {
-            output: {
-                item: 'emendatusenigmatica:dimensional_chunk'
-            },
+            output: { item: 'emendatusenigmatica:dimensional_chunk' },
             rarity: [
                 {
                     whitelist: endBiomes,
                     blacklist: {},
-                    depth_min: 0,
-                    depth_max: 255,
+                    depth_min: 30,
+                    depth_max: 60,
```

Comment:
what's the reasoning behind this change? :)

Reply:
Just seemed like the focused version (higher weight). Should be a narrower range now that there's actually more than one ore in the end.

Reply:
Ah makes sense 👍